packages/suite/src/middlewares/wallet/tradingMiddleware.ts

```diff
@@ -86,14 +87,6 @@ export const tradingMiddleware =
                     );
                 }
```

Comment:
Please verify if the getInfo (the HTTP request) is not called twice due to getInfo called in the new load initial data. And potentially fix it so there is only single HTTP request at time.